packages/stylelint-plugin/src/stylelintPlugin.spec.ts

```diff
@@ -221,23 +245,29 @@ const testExamples: TestExample[] = [
         another: rgb(@color-blue-200-rgb);
         foo: rgba(@color-purple-700-rgb, 0.9);
       }`,
-    expectedWarnings: 1,
+    expectedWarningMessages: [
+      replacementCssVariableUsedWithinUnsupportedFunction(
+        "kz-color-cluny-700",
+        "color-blue-700",
+        "darken"
+      ),
+    ],
   },
   {
     language: "scss",
     testName: "doesn't fix tokens within variables",
     input:
-      '@import "~@kaizen/design-tokens/sass/color"; $foo: $kz-color-wisteria-800;',
+      '@import "~@kaizen/design-tokens/sass/color"; $foo: $color-purple-800;',
```

Comment:
Is the input supposed to change on this test?

I imagine you still want the old `kz-color-wisteria-800` token so that you can test it isn't fixed. Either that, or the `testName` feels misleading and I'm misunderstanding what the test is supposed to be :)

Reply:
Hmm this is a good point thanks, it probably overlooked it when renaming things.